sesman/startwm.sh

```diff
@@ -3,6 +3,8 @@
 #start the window manager
 wm_start()
 {
+  mcookie | sed -e 's/^/add '$DISPLAY' . /' | xauth -q
```

Comment:
I don't like to depend on mcookie command because operating systems other than GNU/Linux doesn't have such a command.  Please be careful of portability. 

I worked on similar issue for TigerVNC.  See also TigerVNC/tigervnc#366.

This is a little bit longer but can replace mcookie command.
`dd if=/dev/urandom bs=16 count=1 2>/dev/null| od -x -An | tr -d ' '`